frontend/sass/_modal.scss

```diff
@@ -22,21 +22,52 @@
   flex-direction: column;
 }
 
+// This styling applies to desktop Internet Explorer
+.scrollable {
+  overflow-y: auto;
+  max-width: 640px;
+}
+
 .jf-modal-dialog {
   cursor: default;
+  overflow-y: initial !important;
+  max-width: 90vw;
 }
 
-.modal-content {
-  // This appears to be the only way to size our modal on mobile
-  // devices in a way that works on *both* IE11 and modern browsers.
-  width: calc(100vw - 40px);
+// This styling applies to mobile browsers
+@media screen and (max-width: $tablet) {
+  .scrollable {
+    max-height: 90vh;
+    max-width: 90vw;
+  }
 }
 
-// This is just a recapitulation of Bulma's default styling, which
-// we apparently need to repeat because otherwise the above
-// .modal-content rule will override it.
+// This styling seems to apply to every desktop browser other than Internet Explorer
 @media screen and (min-width: $tablet) {
-  .modal-content {
-    width: 640px;
+  .scrollable {
+    height: auto;
+    max-height: 90vh;
+    max-width: 75vw;
+  }
+  .jf-modal-dialog {
+    position: absolute;
+    max-width: 75vw;
+    left: 28%;
+    left: calc(50% - 320px);
+    top: 15%;
+  }
+}
+// This styling applies when a desktop browser is very short vertically
+@media screen and (max-height: 400px) {
+  .scrollable {
+    height: 250px;
+    max-width: 75vw;
+  }
+  .jf-modal-dialog {
+    position: absolute;
+    max-width: 75vw;
+    left: 28%;
+    left: calc(50% - 320px);
+    top: 15%;
   }
```

Comment:
It seems like there's a lot of repetition between this rule and another one about 10 lines above it.  Do you think it can be refactored to make things more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), either via standard CSS or SCSS (like a [mixin or include](https://sass-lang.com/documentation/at-rules/mixin))?